src/components/Comments/WriteComment.js

```diff
@@ -9,15 +9,15 @@ import {
   TouchableOpacity,
   View
 } from 'react-native';
-import ToggleSwitch from '../ToggleSwitch';
 
 import AnonUserInfoRepo from '../../service/repo/anonUserInfoRepo';
+import SendIcon from '../SendIcon';
 import StringConstant from '../../utils/string/StringConstant';
+import ToggleSwitch from '../ToggleSwitch';
+import {CHAT_ANON, CHAT_SIGNED} from '../../utils/constants';
+import {COLORS} from '../../utils/theme';
 import {Context} from '../../context';
 import {fonts, normalize} from '../../utils/fonts';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statements have been rearranged. As long as all the necessary modules are still imported, this change should not affect the functionality of the code. However, it's a good practice to keep imports organized either alphabetically or by their purpose (e.g., library imports, component imports, utility imports, etc.) for better readability and maintainability.
```diff
- import ToggleSwitch from '../ToggleSwitch';
- import AnonUserInfoRepo from '../../service/repo/anonUserInfoRepo';
- import StringConstant from '../../utils/string/StringConstant';
- import {Context} from '../../context';
- import {fonts, normalize} from '../../utils/fonts';
- import {COLORS} from '../../utils/theme';
- import SendIcon from '../SendIcon';
- import {CHAT_ANON, CHAT_SIGNED} from '../../utils/constants';

+ import AnonUserInfoRepo from '../../service/repo/anonUserInfoRepo';
+ import SendIcon from '../SendIcon';
+ import StringConstant from '../../utils/string/StringConstant';
+ import ToggleSwitch from '../ToggleSwitch';
+ import {CHAT_ANON, CHAT_SIGNED} from '../../utils/constants';
+ import {COLORS} from '../../utils/theme';
+ import {Context} from '../../context';
+ import {fonts, normalize} from '../../utils/fonts';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->